lib/analyzer/os-release/static.ts

```diff
@@ -82,5 +82,14 @@ export async function detect(
     osRelease.name = "oracle";
   }
 
+  // Support round version. ie change SLES 15 to SLES 15.0
+  if (
+    osRelease.name.trim() === "sles" &&
+    osRelease.version &&
+    !osRelease.version.includes(".")
+  ) {
+    osRelease.version += ".0";
```

Comment:
Minor: Should we extract it to a function to avoid duplicate code like in the previous file?

Reply:
this entire file will be delete next sprint :) so I don't mind leaving it as is

Reply:
I'm not sure we are deleting it, this our core OS detection logic